src/lib/components/form/BuildForm.svelte

```diff
@@ -225,7 +225,7 @@
 
       const { newBuild } = response;
 
-      await goto(buildPath(newBuild));
+      await goto(buildPath(newBuild), { invalidateAll: true });
```

Comment:
Why did this need to be invalidate?

Reply:
Because the old version of the build is displayed when submitting an edit form. Ideally I would've liked to invalidate only the build path, but for some reason that doesn't trigger a full round-trip to the server to fetch the new data.